lib/y2logsstep.pm

```diff
@@ -209,13 +209,16 @@ sub deal_with_dependency_issues {
     }
 }
 
-sub verify_license_has_to_be_accepted {
-    # license+lang
+sub accept_license {
+    my ($self, $verify_license_has_to_be_accepted) = @_;
```

Comment:
I believe this variable name is too long. What's about something like `extended_check` or at least skipping word license, as it's clear from the subroutine name.
Also, could you please document this method using POD or just comment.